### ISSUE TITLE
Provide a catkin-free cmake file

### ISSUE THREAD

**Issue by grcasanova** [ethz-asl/grid_map#130]
As grid_map core does not need ROS I would add a custom Cmake file without catkin references for those who don't need ROS and don't have it installed.

**Comment by pfankhauser** [comment]
Good idea, @grcasanova! If you can work on this, I'd be happy to integrate it.
